### PR TITLE
Move simple proofreading rules into round.php

### PR DESCRIPTION
**Pull request by cpeel**
As part of the post-release cleanup of `faq/`, move the simple proofreading guidelines into the round page.

Sandbox: https://www.pgdp.org/~cpeel/c.branch/slurp-simple-proof-rules/tools/proofers/round.php?round_id=P1&numofpages=81

### CODE REVIEW

**Comment by bpfoley** [comment]
All the `locale/*/LC_MESSAGES/messages.po` have comments referencing the old `.inc` file. Do those need to get updated?

```
971:#: pinc/simple_proof_text.inc:6
975:#: pinc/simple_proof_text.inc:9
979:#: pinc/simple_proof_text.inc:11
983:#: pinc/simple_proof_text.inc:13
987:#: pinc/simple_proof_text.inc:15
991:#: pinc/simple_proof_text.inc:17
995:#: pinc/simple_proof_text.inc:19
999:#: pinc/simple_proof_text.inc:24
1004:#: pinc/simple_proof_text.inc:26
```

**Comment by cpeel** [comment]
> All the `locale/*/LC_MESSAGES/messages.po` have comments referencing the old `.inc` file. Do those need to get updated?

No, we update the `messages.po` files wholesale when we have new translations to add not when we modify strings (or remove files).

**Review by bpfoley — APPROVED** [review]

**Review by srjfoo — APPROVED** [review]

**Comment by srjfoo** [comment]
> > All the `locale/*/LC_MESSAGES/messages.po` have comments referencing the old `.inc` file. Do those need to get updated?
> 
> No, we update the `messages.po` files wholesale when we have new translations to add not when we modify strings (or remove files).

Once the code is merged and deployed, and the PO Template regenerated, any strings that are no longer in the active code will be disabled (they actually stay in the template unless someone "purges deleted strings" (that's poedit's terminology)). (nts: add "regenerate POT" to draft deployment notes)

**Review by 70ray — APPROVED** [review]

File: tools/proofers/round.php [hunk 1]
```diff
@@ -43,17 +43,39 @@
...
+    echo "<br><br>";
+    echo _("3) Remove extra spaces around punctuation mistakenly inserted by the OCR software.");
+    echo "<br><br>";
+    echo _("4) Don't correct the original spelling.");
+    echo "<br><br>";
+    // TRANSLATORS: PM = project manager
+    echo _("5) When in doubt, make it look like the original and use [** <i>Notes for the next proofreader or PM would go here</i>] to flag the spot.");
+    echo "</p>\n\n";
+
+
```
[the remark this answers is not in the document]
Comment (cpeel) [reply tools/proofers/round.php:73]: I'm OK with the extra space for now 😁

**Review by chrismiceli — APPROVED** [review]

**Pull request merged**